packages/next/types/index.d.ts

```diff
@@ -37,12 +37,6 @@ declare module 'react' {
   interface LinkHTMLAttributes<T> extends HTMLAttributes<T> {
     nonce?: string
   }
-
-  // <style jsx> and <style jsx global> support for styled-jsx
-  interface StyleHTMLAttributes<T> extends HTMLAttributes<T> {
-    jsx?: boolean
-    global?: boolean
```

Comment:
Didn't know there were `styled-jsx` types inside `next`, even if apparently it was the case, it was not working, users still needed to install `@types/styled-jsx`, now that it is included in `styled-jsx@3.4.1`, we don't need it anymore inside `next`.